tests/dummy/app/controllers/components-examples/flexberry-menu/settings-example.js

```diff
@@ -32,26 +32,40 @@ export default Ember.Controller.extend({
 
     @property items
     @type Object
-   */
+  */
```

Comment:
Yep, there are no spaces in such lines in ember source code: https://github.com/emberjs/ember.js/blob/v2.6.0/packages/ember-runtime/lib/system/string.js#L153

I don't like it too, but if we decided to follow ember-like code style, ok then, space is unnecessary here.